<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="d_get_privatekeyvalue">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txt_subtitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Get your keys or enter your own below</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_get_publickeyvalue">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txt_publickey_n">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>n :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_public_key">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PUBLIC KEY</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_private_key">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PRIVATE KEY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_get_publickeyvalue">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txt_privatekey_d">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>d :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_publickey_e">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>e :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_getkey">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Get keys</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_get_privatekeyvalue">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txt_title">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DSA Cipher</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_privatekey_n">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>n :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_loadfile">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Load file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_signfile">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign file</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_file">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>File :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_verification">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>690</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Verify</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_verification">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>690</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Is the signature valid ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_signature">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>590</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sign file :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_type">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_size">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Size :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_location">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>500</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Location :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_fileinfo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>540</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_signature">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>590</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txt_filevalue">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="txt_typevalue">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="txt_sizevalue">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="txt_locationvalue">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>500</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
